metals/src/main/scala/scala/meta/internal/bsp/BspConnector.scala

```diff
@@ -2,6 +2,7 @@ package scala.meta.internal.bsp
 
 import java.nio.file.Files
 
+import scala.collection.immutable.Nil
```

Comment:
Huh interesting. I imported it earlier to use it, but then ended up not needing it. However, I'm surprised this wasn't removed with Scalafix. I wonder if it's because we use `Nil` in the pattern match?

Reply:
I think the symbol is used, but also is in Predef, so usually it's just implictely imported.